src/components/KTextArea/KTextArea.vue

```diff
@@ -18,16 +18,34 @@
       </template>
     </KLabel>
 
-    <textarea
-      :id="textAreaId"
-      v-bind="modifiedAttrs"
-      :aria-invalid="ariaInvalid"
-      class="input-textarea"
-      :class="[resizable || isResizable ? 'resizable' : undefined]"
-      :rows="rows"
-      :value="getValue()"
-      @input="inputHandler"
-    />
+    <!--
+      The wrapper element is required for the grid fallback for browsers that don’t support `field-sizing: content`.
+      The data-value attribute is used here to render the textarea content in the grid fallback. It’s rendered via
+      a CSS pseudo-element with `content: attr(data-value) " "`, ensuring it’s at least one line tall.
+      See https://chriscoyier.net/2023/09/29/css-solves-auto-expanding-textareas-probably-eventually/
+    -->
+    <div
+      class="input-textarea-wrapper"
+      :class="{
+        autosize,
+        legacy: autosize && !SUPPORT_FIELD_SIZING_CONTENT
```

Comment:
![image](https://github.com/user-attachments/assets/88821866-4b47-4aaa-b045-cdeb1c71c867)
issue: the current implementation functionally works in SSR; however, it causes hydration errors

Reply:
The mismatch is expected so I think we'd just allow it. I'll add  a `data-allow-mismatch="class"` annotation here.

Reply:
Works for me